### PR TITLE
JAIS-270 feat: figma import

### PR DESCRIPTION
**Pull request by comictvn**
## Description

Support sending figma information to AI

## Issues

https://jitera.atlassian.net/browse/JAIS-270

## Videos

Video to prove your work

## Test

What kind of tests were performed and what were the results?

# Checklist

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation to reflect the change
- [ ] I have updated the README.md to reflect the change
- [ ] I have added unit tests to prove that the update is effective and the feature works

### CODE REVIEW

**Comment by maxyzx** [comment]
Did you publish a new node version?

File: protos/v2/project_figma.proto [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	int32 project_id = 2;
+	string access_token = 3;
+	string node_ids = 4;
+	string file_key = 5;
+}
+
+message ProjectFigmaImportResponse {
+	int32 project_generate_queue_id = 1;
+	int32 project_id = 2;
+  Module module = 3;
```
Comment (zrg-team) [inline protos/v2/project_figma.proto:17]: format it correctly

File: protos/v2/project_figma.proto [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  repeated Block use_case_blocks = 6;
+	string error_message = 7;
+
+	enum Module {
+		FIGMA_TO_NATURAL_LANGUAGE = 0;
+	}
+
+	enum Status {
+		SUCCEEDED = 0;
+		INPROGRESS = 1;
```
Comment (zrg-team) [inline protos/v2/project_figma.proto:29]: @comictvn When some node_ids batch fail ? how i report what is the failded to Jitera Core ?

Comment (comictvn) [reply protos/v2/project_figma.proto:29]: @zrg-team there isn't confirm requirement it yet. when have UI or spec we can modify it.

**Comment by comictvn** [comment]
@maxyzx @zrg-team addressed feedback

**Review by zrg-team — APPROVED** [review]

**Review by maxyzx — APPROVED** [review]

**Review by akuita — APPROVED** [review]

**Pull request merged**